CONTRIBUTING.md

```diff
@@ -0,0 +1,209 @@
+# Contributing to BEEP
+
+We are striving for community-driven adoption and development of BEEP. If you would like to 
+contribute (thank you!), please have a look at the [guidelines below](#workflow).
+
+If you're already familiar with our workflow, maybe have a quick look at the [pre-commit checks](#pre-commit-checks) directly below.
+
+## Pre-commit checks
+
+Before you commit any code, please perform the following checks:
+
+- [All tests pass](#testing): `$ nosetests beep`
```

Comment:
I'd recommend pytest instead, since nosetests is a little out of date and we're using pytest in the CI.

Reply:
I didn't realize. I will also update the readme to reflect the same.

Reply:
It looks like the readme still needs the `nosetests` -> `pytest` update.